packages/yoastseo/src/values/Paper.js

```diff
@@ -45,14 +44,19 @@ var Paper = function( text, attributes ) {
 		attributes.locale = defaultAttributes.locale;
 	}
 
+	if ( attributes.hasOwnProperty( "url" ) ) {
+		console.warn( "The 'url' attribute is deprecated, use 'slug' instead." );
```

Comment:
Can we add some more information about when it has been deprecated and/or when it will be removed from the code? 

In [our PHP docs on how to deprecate code](https://yoast.atlassian.net/wiki/spaces/PLUGIN/pages/1194360977/How+to+deprecate+functionality+in+Yoast+plugins) we always add the version number in which the method / function / property has been deprecated.

Reply:
Good point, I'll see how to best document the deprecation of this attribute. The `getUrl` and `hasUrl`-function do have `@deprecated` documentation.

Reply:
I now added a comment in the code. AFAIK JavaScript does not provide a way to deprecate an attribute through the `@deprecated` annotation. But I'd love to know if there's a way 😄